File: launchdarkly/csharp/src/OSEG.LaunchDarklyExamples/ResetEnvironmentSDKKeyExample.cs
using System;
using System.Collections.Generic;
using System.IO;
using System.Text.Json;

using Org.LaunchDarklyTools.Api;
using Org.LaunchDarklyTools.Client;
using Org.LaunchDarklyTools.Model;

namespace OSEG.LaunchDarklyExamples;

public class ResetEnvironmentSDKKeyExample
{
    public static void Run()
    {
        var config = new Configuration();
        config.ApiKey = new Dictionary<string, string> {["ApiKey"] = "YOUR_API_KEY"};

        try
        {
            var response = new EnvironmentsApi(config).ResetEnvironmentSDKKey(
                projectKey: null,
                environmentKey: null,
                expiry: null
            );

            Console.WriteLine(response);
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Environments#ResetEnvironmentSDKKey: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}

File: launchdarkly/typescript-axios/src/ResetEnvironmentSDKKeyExample.ts
import axios, { AxiosError } from "axios";
import * as api from "launchdarkly_client"

const configuration = new api.Configuration({
  apiKey: "YOUR_API_KEY",
});

new api.EnvironmentsApi(configuration).resetEnvironmentSDKKey(
  "projectKey_string", // projectKey
  "environmentKey_string", // environmentKey
  undefined, // expiry
).then(response => {
  console.log(response.data);
}).catch((error: Error | AxiosError) => {
  console.log("Exception when calling EnvironmentsApi#resetEnvironmentSDKKey:");
  axios.isAxiosError(error) ? console.log(error.message) : console.log(error);
});
